waspc/data/Generator/templates/server/src/jobs/_job.ts

```diff
@@ -1,12 +1,12 @@
 {{={= =}=}}
 import { createJob } from './{= jobExecutorRelativePath =}'
```

Comment:
As agreed, this stays a partial import until `JsImport` starts supporting types.

Reply:
@sodic We are not importing a type here?

Reply:
I wanted to comment it in a different place.